packages/frontend/src/api/useParties.ts

```diff
@@ -149,5 +161,7 @@ export const useParties = (): UsePartiesOutput => {
     currentEndUser: data?.parties.find((party) => party.isCurrentEndUser),
     deletedParties: data?.deletedParties ?? [],
     allOrganizationsSelected,
+    isCompanyProfile,
```

Comment:
drop `isCompanyProfile` and rather pass `selectedProfile === 'company'` to `Background`, or better (?), change prop to `Background` to be `profile` instead?